DESCRIPTION

```diff
@@ -37,6 +37,8 @@ Suggests:
     rmarkdown,
     shiny (>= 1.2.0),
     testit
+Remotes:
+    rstudio/shiny
```

Comment:
Is there a plan for the next CRAN release of **shiny**? I'm wondering roughly when we could remove this `Remotes` field in **DT** (in weeks or months), so the release of **DT** won't be blocked. Thanks!

Reply:
We're planning to release in early January. The only reason that this Remotes entry is needed is because we link to the docs for a new function in Shiny. If you need to release earlier, we can remove that link and put it in a PR which we can merge later.

Reply:
Got you. Thanks!